tests/integration/utils.py

```diff
@@ -0,0 +1,34 @@
+"""Integration test utils."""
+import time
+
+from solana.rpc.api import Client
+from solana.rpc.types import RPCResponse
+
+
+def assert_valid_response(resp: RPCResponse):
+    """Assert valid RPCResponse."""
+    assert resp["jsonrpc"] == "2.0"
+    assert resp["id"]
+    assert resp["result"]
+
+
+def confirm_transaction(client: Client, tx_sig: str) -> RPCResponse:
+    """Confirm a transaction."""
+    TIMEOUT = 30  # 30 seconds  pylint: disable=invalid-name
+    elapsed_time = 0
+    while elapsed_time < TIMEOUT:
+        sleep_time = 3
```

Comment:
So the transaction time should be three seconds after the initial sleep period. Are the `7` seconds or `3` seconds based on average Solana transaction confirmation time?